config/eye.yml.erb

```diff
@@ -0,0 +1,65 @@
+<% app_root = "/home/nucore/nucore.stage.tablexi.com/current" %>
+
+name: "nucore_open_stage"
+
+config:
+  logger: <%= app_root %>/log/eye.log
+
+application:
+  working_dir: <%= app_root %>
+  stop_on_delete: true
+
+triggers:
+  - name: flapping
+    config:
+      times: 5
+      within: 5 minutes
+
+notifications:
+  - name: monitoring
+    type: ses
+    level: info
+    contact: nucore+open@tablexi.com
+    config:
+      from: nucore+open@tablexi.com
+      access_key_id: ~
+      secret_access_key: ~
```

Comment:
This should get converted over to use datadog https://www.notion.so/tablexi/Eye-b994f9d3c11c412c9bdfc8c2dacf5fa6#dbc6377e9ced43c9b87cac80e5b4d5f1